src/SFA.DAS.AANHub.Application/Partners/Queries/GetPartnerMemberResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Partners.Queries
+{
+    public class GetPartnerMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string? Email { get; set; }
```

Comment:
I've just got a similar comment on my recent PR, but make these non-nullable and just initialise the property with ` = null!;`

Reply:
Fixed